spec/presenters/post_presenter_spec.rb

```diff
@@ -135,6 +135,14 @@
       presenter = PostPresenter.new(status_message_with_poll)
       expect(presenter.as_json).to be_a(Hash)
     end
+
+    it "returns the answer id of the current user's poll participation" do
+      presenter = PostPresenter.new(status_message_with_poll, alice)
+      poll_answer = status_message_with_poll.poll.poll_answers.first
+      poll_participation = status_message_with_poll.poll.poll_participations
+      poll_participation = poll_participation.create(poll_answer: poll_answer, author: alice.person)
+      expect(presenter.poll_participation_answer_id).to eql(poll_participation.poll_answer_id)
+    end
```

Comment:
When you do the change in `post_presenter_spec.rb` this would also need a negative test to test it returns nil when the user didn't participate yet.